crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -345,32 +343,32 @@ where
         key: Self::Key,
         layout: &MeshVertexBufferLayout,
     ) -> Result<RenderPipelineDescriptor, SpecializedMeshPipelineError> {
-        let mut bind_group_layouts = vec![if key
-            .mesh_key
-            .contains(MeshPipelineKey::MOTION_VECTOR_PREPASS)
-        {
-            self.view_layout_motion_vectors.clone()
-        } else {
-            self.view_layout_no_motion_vectors.clone()
-        }];
+        let mesh_key = key.mesh_key;
```

Comment:
Maybe add a similar note here that the non prepass version should be updated too if necessary.